File: java/Baju.java
public class Baju {
    private String untuk, size, merk;

    public Baju() {
        this.untuk = "";
        this.size = "";
        this.merk = "";
    }

    public Baju(String untuk, String size, String merk) {
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public String getUntuk() {
        return untuk;
    }

    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getMerk() {
        return merk;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }
}

File: php/Baju.php
<?php

class Baju {
    // attributes
    private $untuk = "";
    private $size = "";
    private $merk = "";

    // constructor
    public function __construct($untuk = "", $size = "", $merk = "") {
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    // getter dan setter
    public function get_untuk() {
        return $this->untuk;
    }

    public function set_untuk($untuk) {
        $this->untuk = $untuk;
    }

    public function get_size() {
        return $this->size;
    }

    public function set_size($size) {
        $this->size = $size;
    }

    public function get_merk() {
        return $this->merk;
    }

    public function set_merk($merk) {
        $this->merk = $merk;
    }
}

?>